source/dub/compilers/buildsettings.d

```diff
@@ -16,9 +16,12 @@ import std.typecons : BitFlags;
 import std.algorithm.iteration : uniq;
 import std.range : chain;
 
+enum timeoutExitStatus = 11111; // exit status for process when timed out
```

Comment:
This should be some carefully chosen value. Do you have any suggestions, @thewilsonator and @CyberShadow?

Reply:
The `timeout` tool uses 124.

Reply:
Adjusted at 28c516458484c5520d2014bd4921e5bcd1f40696.